components/automate-ui/src/app/components/resource-dropdown/resource-dropdown.component.ts

```diff
@@ -66,15 +66,16 @@ export class ResourceDropdownComponent implements OnInit, OnChanges {
     return ChefSorters.naturalSort(this.resources, 'name');
   }
 
-  toggleDropdown(event: MouseEvent): void {
-    event.stopPropagation();
```

Comment:
Remove the suppression on event propagation--we need that to percolate when other dropdowns are present. (Again, see preamble.)